pages/k8s/storage.md

```diff
@@ -191,7 +192,7 @@ Make use of **Juju** constraints to allocate an instance with the required amoun
 juju deploy nfs --constraints root-disk=200G
 ```
 
-### Relate to CDK
+### Relate to harmed Kubernetes
```

Comment:
I, too, can relate to being harmed by Kubernetes  ;)

Reply:
:+1:  :smile: